### PR TITLE
[Dungeon] Razorfen Downs

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.
Escort NPC (For quest) updated. It was imposible to catch up.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1588337666335849000.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+UPDATE `creature_template` SET `mindmg` = 96, `maxdmg` = 129, `DamageModifier` = 1.03 WHERE `entry` = 7346;
+UPDATE `creature_template` SET `mindmg` = 134, `maxdmg` = 180, `DamageModifier` = 1.03 WHERE `entry` = 7345;
+UPDATE `creature_template` SET `mindmg` = 46, `maxdmg` = 62, `DamageModifier` = 1.03 WHERE `entry` = 8585;
+
+/* RARE */
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 140, `maxdmg` = 158, `DamageModifier` = 1.02 WHERE `entry` = 7354;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 208, `maxdmg` = 268, `DamageModifier` = 1.01 WHERE `entry` = 7355;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 208, `maxdmg` = 268, `DamageModifier` = 1.01 WHERE `entry` = 7357;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588337666335849000.sql:37]:
```suggestion
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 208, `maxdmg` = 268, `DamageModifier` = 1.01 WHERE `entry` IN (7355, 7357);
```

File: data/sql/updates/pending_db_world/rev_1588337666335849000.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+/*
+ * Dungeon: Razorfen Downs
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
+*/
+
+/* REGULAR */ 
+UPDATE `creature_template` SET `mindmg` = 90, `maxdmg` = 125, `DamageModifier` = 1.03 WHERE `entry` = 7332;
+UPDATE `creature_template` SET `mindmg` = 90, `maxdmg` = 125, `DamageModifier` = 1.03 WHERE `entry` = 7327;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588337666335849000.sql:11]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 90, `maxdmg` = 125, `DamageModifier` = 1.03 WHERE `entry` IN (7327, 7332);
```

File: data/sql/updates/pending_db_world/rev_1588337666335849000.sql [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+UPDATE `creature_template` SET `mindmg` = 93, `maxdmg` = 129, `DamageModifier` = 1.03 WHERE `entry` = 7334;
+UPDATE `creature_template` SET `mindmg` = 93, `maxdmg` = 129, `DamageModifier` = 1.03 WHERE `entry` = 7329;
+UPDATE `creature_template` SET `mindmg` = 130, `maxdmg` = 180, `DamageModifier` = 1.03 WHERE `entry` = 7328;
+UPDATE `creature_template` SET `mindmg` = 55, `maxdmg` = 74, `DamageModifier` = 1.03 WHERE `entry` = 7349;
+UPDATE `creature_template` SET `mindmg` = 170, `maxdmg` = 230, `DamageModifier` = 1.03 WHERE `entry` = 7351;
+UPDATE `creature_template` SET `mindmg` = 93, `maxdmg` = 125, `DamageModifier` = 1.03 WHERE `entry` = 7343;
+UPDATE `creature_template` SET `mindmg` = 77, `maxdmg` = 104, `DamageModifier` = 1.03 WHERE `entry` = 7352;
+UPDATE `creature_template` SET `mindmg` = 99, `maxdmg` = 129, `DamageModifier` = 1.03 WHERE `entry` = 7348;
+UPDATE `creature_template` SET `mindmg` = 59, `maxdmg` = 84, `DamageModifier` = 1.03 WHERE `entry` = 7341;
+UPDATE `creature_template` SET `mindmg` = 93, `maxdmg` = 129, `DamageModifier` = 1.03 WHERE `entry` = 7344;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588337666335849000.sql:27]:
```suggestion
UPDATE `creature_template` SET `mindmg` = 93, `maxdmg` = 129, `DamageModifier` = 1.03 WHERE `entry` IN (7334, 7329, 7344);
UPDATE `creature_template` SET `mindmg` = 130, `maxdmg` = 180, `DamageModifier` = 1.03 WHERE `entry` = 7328;
UPDATE `creature_template` SET `mindmg` = 55, `maxdmg` = 74, `DamageModifier` = 1.03 WHERE `entry` = 7349;
UPDATE `creature_template` SET `mindmg` = 170, `maxdmg` = 230, `DamageModifier` = 1.03 WHERE `entry` = 7351;
UPDATE `creature_template` SET `mindmg` = 93, `maxdmg` = 125, `DamageModifier` = 1.03 WHERE `entry` = 7343;
UPDATE `creature_template` SET `mindmg` = 77, `maxdmg` = 104, `DamageModifier` = 1.03 WHERE `entry` = 7352;
UPDATE `creature_template` SET `mindmg` = 99, `maxdmg` = 129, `DamageModifier` = 1.03 WHERE `entry` = 7348;
UPDATE `creature_template` SET `mindmg` = 59, `maxdmg` = 84, `DamageModifier` = 1.03 WHERE `entry` = 7341;
```

File: data/sql/updates/pending_db_world/rev_1588337666335849000.sql [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+UPDATE `creature_template` SET `mindmg` = 46, `maxdmg` = 62, `DamageModifier` = 1.03 WHERE `entry` = 8585;
+
+/* RARE */
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 140, `maxdmg` = 158, `DamageModifier` = 1.02 WHERE `entry` = 7354;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 208, `maxdmg` = 268, `DamageModifier` = 1.01 WHERE `entry` IN (7355, 7357);
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 229, `maxdmg` = 295, `DamageModifier` = 1.01 WHERE `entry` = 8567;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 215, `maxdmg` = 276, `DamageModifier` = 1.01 WHERE `entry` = 7356;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 269, `maxdmg` = 346, `DamageModifier` = 1.01 WHERE `entry` = 7358;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588337666335849000.sql:36]:
```suggestion
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 208, `maxdmg` = 268, `DamageModifier` = 1.01 WHERE `entry` IN (7355, 7357);
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 229, `maxdmg` = 295, `DamageModifier` = 1.01 WHERE `entry` = 8567;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 215, `maxdmg` = 276, `DamageModifier` = 1.01 WHERE `entry` = 7356;
UPDATE `creature_template` SET `type_flags`=`type_flags`|4, `mindmg` = 269, `maxdmg` = 346, `DamageModifier` = 1.01 WHERE `entry` = 7358;
```

**Comment by Geekena** [comment]
damage and spells are ok
didnt have any problem in this one

**Comment by Kitzunu** [comment]
Ty for the PR!

**Pull request merged**